factory/config.go

```diff
@@ -79,29 +79,29 @@ type Configuration struct {
 	UserPlaneInformation UserPlaneInformation `yaml:"userplane_information"`
 	ServiceNameList      []string             `yaml:"serviceNameList,omitempty"`
 	SNssaiInfo           []SnssaiInfoItem     `yaml:"snssaiInfos,omitempty"`
-	ULCL                 bool                 `yaml:"ulcl,omitempty"`
 	EnterpriseList       map[string]string    `yaml:"enterpriseList,omitempty"`
-	EnableDbStore        bool                 `yaml:"enableDBStore,omitempty"`
-	EnableUpfAdapter     bool                 `yaml:"enableUPFAdapter,omitempty"`
 	SmfDbName            string               `yaml:"smfDBName,omitempty"`
 	Mongodb              *Mongodb             `yaml:"mongodb,omitempty"`
 	KafkaInfo            KafkaInfo            `yaml:"kafkaInfo,omitempty"`
-	DebugProfilePort     int                  `yaml:"debugProfilePort,omitempty"`
 
-	EnableNrfCaching         bool           `yaml:"enableNrfCaching"`
-	NrfCacheEvictionInterval int            `yaml:"nrfCacheEvictionInterval"`
 	StaticIpInfo             []StaticIpInfo `yaml:"staticIpInfo"`
+	NrfCacheEvictionInterval int            `yaml:"nrfCacheEvictionInterval"`
+	DebugProfilePort         int            `yaml:"debugProfilePort,omitempty"`
+	EnableNrfCaching         bool           `yaml:"enableNrfCaching"`
+	EnableDbStore            bool           `yaml:"enableDBStore,omitempty"`
+	EnableUpfAdapter         bool           `yaml:"enableUPFAdapter,omitempty"`
+	ULCL                     bool           `yaml:"ulcl,omitempty"`
```

Comment:
Why not keeping variables together based on their type (when possible)) such as below?
```diff
diff --git a/factory/config.go b/factory/config.go
index 7ca35d2..a1e803f 100644
--- a/factory/config.go
+++ b/factory/config.go
 type Configuration struct {
-       SmfName              string               `yaml:"smfName,omitempty"`
-       Sbi                  *Sbi                 `yaml:"sbi,omitempty"`
-       PFCP                 *PFCP                `yaml:"pfcp,omitempty"`
-       NrfUri               string               `yaml:"nrfUri,omitempty"`
-       UserPlaneInformation UserPlaneInformation `yaml:"userplane_information"`
-       ServiceNameList      []string             `yaml:"serviceNameList,omitempty"`
-       SNssaiInfo           []SnssaiInfoItem     `yaml:"snssaiInfos,omitempty"`
-       ULCL                 bool                 `yaml:"ulcl,omitempty"`
-       EnterpriseList       map[string]string    `yaml:"enterpriseList,omitempty"`
-       EnableDbStore        bool                 `yaml:"enableDBStore,omitempty"`
-       EnableUpfAdapter     bool                 `yaml:"enableUPFAdapter,omitempty"`
-       SmfDbName            string               `yaml:"smfDBName,omitempty"`
-       Mongodb              *Mongodb             `yaml:"mongodb,omitempty"`
-       KafkaInfo            KafkaInfo            `yaml:"kafkaInfo,omitempty"`
-       DebugProfilePort     int                  `yaml:"debugProfilePort,omitempty"`
-
-       EnableNrfCaching         bool           `yaml:"enableNrfCaching"`
-       NrfCacheEvictionInterval int            `yaml:"nrfCacheEvictionInterval"`
-       StaticIpInfo             []StaticIpInfo `yaml:"staticIpInfo"`
+       Mongodb                  *Mongodb             `yaml:"mongodb,omitempty"`
+       PFCP                     *PFCP                `yaml:"pfcp,omitempty"`
+       Sbi                      *Sbi                 `yaml:"sbi,omitempty"`
+       NrfUri                   string               `yaml:"nrfUri,omitempty"`
+       SmfName                  string               `yaml:"smfName,omitempty"`
+       SmfDbName                string               `yaml:"smfDBName,omitempty"`
+       SNssaiInfo               []SnssaiInfoItem     `yaml:"snssaiInfos,omitempty"`
+       StaticIpInfo             []StaticIpInfo       `yaml:"staticIpInfo"`
+       ServiceNameList          []string             `yaml:"serviceNameList,omitempty"`
+       EnterpriseList           map[string]string    `yaml:"enterpriseList,omitempty"`
+       KafkaInfo                KafkaInfo            `yaml:"kafkaInfo,omitempty"`
+       UserPlaneInformation     UserPlaneInformation `yaml:"userplane_information"`
+       DebugProfilePort         int                  `yaml:"debugProfilePort,omitempty"`
+       NrfCacheEvictionInterval int                  `yaml:"nrfCacheEvictionInterval"`
+       EnableDbStore            bool                 `yaml:"enableDBStore,omitempty"`
+       EnableNrfCaching         bool                 `yaml:"enableNrfCaching"`
+       EnableUpfAdapter         bool                 `yaml:"enableUPFAdapter,omitempty"`
+       ULCL                     bool                 `yaml:"ulcl,omitempty"`
 }
```

Reply:
done